File: app/templates/server/go/echo/api/todo/route/todoroute.go
package todoroute

import (
	"<%= repoHostUrl %>/<%= userNameSpace %>/<%= appName %>/server/api/todo/controller"
	"gopkg.in/labstack/echo.v2"
)

func Init(e *echo.Echo) {
	e.GET("/api/todos", todocontroller.GetAll)
	e.GET("/api/todos/:id", todocontroller.GetById)
	e.POST("/api/todos", todocontroller.NewTodo)
	e.DELETE("/api/todos/:id", todocontroller.RemoveTodo)
}

File: app/templates/server/node/express/server_node_express_babel/api/todo/route/todo-route.js
import TodoController from "../controller/todo-controller";

export default class TodoRoutes {
  static init(router) {
    router
    .route("/api/todos")
    .get(TodoController.getAll)
    .post(TodoController.createTodo);

    router
    .route("/api/todos/:id")
    .delete(TodoController.deleteTodo);
  }
}
